logicrunner/logicrunner_test.go

```diff
@@ -2108,6 +2108,8 @@ func (s *LogicRunnerFuncSuite) getObjectInstance(ctx context.Context, am artifac
 }
 
 func TestLogicRunnerFunc(t *testing.T) {
+	t.Skip()
```

Comment:
?

Reply:
LR tests (slowtests) are skipped, until @aalekseev-ru will port them to functests. This PR is postponed until Anton patch will be merged.